Update planned-maintenance.md

tsubasaxZZZ opened the pull request:
Added description of maintenance of kube-system Pod.

tsubasaxZZZ committed:
Update planned-maintenance.md

Added description of maintenance of kube-system Pod.

qpetraroia reviewed (changes requested):
"as well as your kube-system Pods on VMSS instance" 

Can you add an "a" here?

"as well as your kube-system Pods on ***a*** VMSS instance"

other than that, looks good!

tsubasaxZZZ committed:
Update planned-maintenance.md

Before:
"as well as your kube-system Pods on VMSS instance"
After:
"as well as your kube-system Pods on a VMSS instance"
